### PR TITLE
YLP-2967 Fix display of duplicate wizards

### PR DESCRIPTION
**Pull request by celia-rouis**
## Checklist
- [x] PR name contains the Jira ticket number
- [x] Changelog updated (Jira tickets are correctly sorted)
- [x] Code tested (unit tests and integration tests)
- [x] No console errors/warnings in integration tests
- [x] No translation files updated (except for English files)
- [x] Code formatted on every file
- [x] Copyright added on every file
- [x] End-to-end tests successful on CI

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -14,6 +14,7 @@ It is based on Tidepool Blip 1.27.
 
 ### Fixed
 - YLP-2951 Patient data not accessible (infinite loading)
+- YLP-2967 Broken wizard (bolus + meal) display in the Daily view
```
Comment (celia-rouis) [inline CHANGELOG.md:17]: **❔ question:** Version to update? (Decision TBC)‏

Comment (celia-rouis) [reply CHANGELOG.md:17]: Seen with Yasmina: OK for 3.3.0

**Review by ydrisR — APPROVED** [review]
👍

**Pull request merged**